4_advanced_usage/30_technique_creation.txt

```diff
@@ -279,14 +279,14 @@ include::technique-st-sample.xml
 
 ----
 
-====== Define variables
+===== Define variables
```

Comment:
Fixing the title hierarchy here, opened another [issue](http://www.rudder-project.org/redmine/issues/7592) for the content.